### PR TITLE
Fix diff snp

### PR DESCRIPTION
**Pull request by zkamvar**
This one is not quite ready. It addresses #163, but I need input from @smanel on the interpretation of one sentence.

### CODE REVIEW

File: use/DifferentiationSNP.Rmd [hunk 1]
```diff
@@ -72,25 +72,25 @@ faster and keep only 20 SNPs in the object `locus`. We then convert `Mydata1` to
...
-
-locus <- Mydata[, 5:24] 
+locus   <- Mydata[, 5:24] 
 Mydata1 <- df2genind(locus, ploidy = 2, ind.names = ind, pop = population, sep = "")
 Mydata1
 Mydata2 <- genind2hierfstat(Mydata1) 
 ```
 
-## $F_{st}$ (observed and expected heterozygosity)  (package hierfstat)
+## $F_{st}$ (observed and expected heterozygosity)
```
Comment (hlapp) [inline use/DifferentiationSNP.Rmd]: Is it really a good idea to have parenthetical expressions in headings? How about `Observed and expected heterozygosity: $F_{st}$

File: use/DifferentiationSNP.Rmd [hunk 2]
```diff
@@ -149,18 +151,17 @@ remember that we used only 20 SNPs to conduct the analysis.
...
+from hierarchical $F_{st}$ analysis (like AMOVA in the case of SNP). We also ran
 a multivariate analysis to investigate the genetic structure of the data at
-individual level assuming no population.
+individual level assuming no population structure.
 
 ### What is next?
 
-You may now want to move to the [estimation of genetic
-distances](VignetteIndividualGeneticDistancesSNP.html)
+You may now want to move to the [estimation of genetic distances](2015-05-18-Dist-SNP.html)
```
Comment (hlapp) [inline use/DifferentiationSNP.Rmd]: _move to_ –> _move on to_. Also add period at end of sentence.

File: use/DifferentiationSNP.Rmd (hunk 1 above)
Comment (zkamvar) [inline use/DifferentiationSNP.Rmd]: :+1:

File: use/DifferentiationSNP.Rmd (hunk 2 above)
Comment (zkamvar) [inline use/DifferentiationSNP.Rmd]: :+1:

**Pull request merged**